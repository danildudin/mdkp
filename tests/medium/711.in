29 7 0
266 849 327 409 684 945 996 987 563 334 241 347 350 763 186 944 331 709 510 225 311 354 371 276 805 563 834 819 623 
898 485 726 556 959 549 886 214 664 787 161 898 210 634 812 832 499 614 827 671 83 103 311 430 980 561 465 583 715 
995 642 430 841 367 628 794 170 840 67 539 920 546 112 990 808 603 724 648 336 466 225 882 776 475 145 842 37 997 
338 945 275 811 59 78 686 449 879 255 916 389 544 326 833 129 651 763 128 317 125 926 917 171 757 370 551 119 947 
63 658 918 655 461 170 750 951 201 353 377 919 337 626 134 298 990 598 458 252 699 193 726 453 675 982 674 784 676 
988 397 356 774 626 542 191 872 613 88 677 886 661 646 565 978 311 400 290 199 34 913 354 732 68 14 720 81 897 
447 119 380 314 613 419 836 564 734 675 40 37 777 567 378 870 147 722 973 855 838 23 246 999 677 55 71 494 193 
890 911 509 715 334 790 5 572 24 900 117 472 553 935 24 272 889 933 208 790 499 791 238 34 294 403 563 546 317 
6735 7149 8421 9068 8895 8582 7260 
