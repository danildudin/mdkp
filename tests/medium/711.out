9707
14
1 5 6 7 10 12 15 17 20 21 24 25 26 27 
